rpc/gnmi/gnmi-specification.md

```diff
@@ -1659,6 +1758,9 @@ limitations under the License
 
 # 7 Revision History
 
+* v0.10.0: May 25, 2023
+  * Add union_replace operation.  Sync revision to gNMI proto revision.
+
```

Comment:
@dplore I am curious, should this PR not be merged before the gnmi.proto is merged? The text here says "sync revision with gnmi proto", but gnmi proto is not even in a PR state"

Reply:
gnmi proto v0.10.0 is now published at: https://github.com/openconfig/gnmi/commit/5473f2ef722ee45c3f26eee3f4a44a7d827e3575#diff-49fc5824be3cfb8d64321cc64142f0568a3039a55da00d2aaed8fd766369854c